bokehjs/test/codebase/size.ts

```diff
@@ -6,12 +6,12 @@ const build_dir = normalize(`${__dirname}/../..`) // build/test/codebase -> buil
 
 const LIMITS = new Map([
   // es2017
-  ["js/bokeh.min.js",                 800],
+  ["js/bokeh.min.js",                 950],
```

Comment:
👀

Reply:
This is just to avoid rebasing between 2.3 and 2.4 due to webgl changes.